src/test/java/ru/training/at/hw7/configs/testng/homework7.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
```

Comment:
I think `resources` is more appropriate place for this file

Reply:
I have moved it to src/test/resources/testng.configs/homework7.xml